src/todo.js

```diff
@@ -0,0 +1,132 @@
+let tasks = [];
+
+function updateIndexes() {
+  tasks.forEach((task, index) => {
+    task.index = index + 1;
+  });
+}
+
+export function saveTasksToLocalStorage() {
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+}
+
+export function deleteTask(todoItem) {
+  todoItem.remove();
+  updateIndexes(); // Update task indexes after deleting a task
+  saveTasksToLocalStorage(); // Save updated tasks to local storage
+}
```

Comment:
- [ ] As mentioned by the previous reviewer, the localStorage is not update when removing a task which make the task displayed when refresh the page even though it has been removed. Kindly check it to follow the project requirement.
### Take a look at this short video :point_down: to understand more
<video src="https://github.com/Hsuwaddyhtat22/To-Do-List-mini-PJ/assets/91142432/60cc0008-1d55-4570-952d-c0beb4b61096" />